migrations/versions/201511231347_225d0750c216_add_contribution_related_tables.py

```diff
@@ -350,15 +350,24 @@ def upgrade():
         sa.Column('phone', sa.String(), nullable=False),
         sa.ForeignKeyConstraint(['event_id'], ['events.events.id']),
         sa.ForeignKeyConstraint(['user_id'], ['users.users.id']),
+        sa.UniqueConstraint('event_id', 'user_id'),
         sa.PrimaryKeyConstraint('id'),
         schema='events'
     )
+    op.create_index(op.f('ix_uq_persons_event_id_email'), 'persons', ['event_id', 'email'], unique=True,
```

Comment:
ah.. probably i kept what alembic autogenerated there. anyway, in that case let's keep it here too and just replace the `op.f(...)` with `None`